tollway/utils.py

```diff
@@ -1,33 +1,23 @@
 import json
-import random
-from datetime import datetime, timedelta
 from pathlib import Path
 from typing import TypedDict
 
-from decouple import config
 from google.cloud import pubsub_v1
 from google.pubsub_v1 import PubsubMessage
 
-from tollway.constants import DATE_VARIATION_MAX, DATE_VARIATION_MIN, TIMESTAMP_FORMAT
+from tollway.constants import PROJECT_ID, TOPIC_ID
 
 
 class EventsLog(TypedDict):
-    past_events_timestamps: list[str]
+    late_events: dict[str, list[str]]
     past_events: list[dict[str, str]]
     all_events: list[dict[str, str]]
 
 
-def get_date_variation(timestamp: str) -> str:
-    random_integer = random.randint(DATE_VARIATION_MIN, DATE_VARIATION_MAX)
-    current_timestamp = datetime.strptime(timestamp, TIMESTAMP_FORMAT)
-    updated_timestamp = (current_timestamp - timedelta(days=random_integer)).strftime(TIMESTAMP_FORMAT)
-    return updated_timestamp
-
-
 def get_topic(pubsub: bool) -> tuple:
     if pubsub:
-        project_id = config("PROJECT_ID")
-        topic_id = config("TOPIC_ID")
+        project_id = PROJECT_ID
```

Comment:
do we need the extra variables? I would probably use the constants straight away

Reply:
yes, great catch. I've overlooked this. Thanks.